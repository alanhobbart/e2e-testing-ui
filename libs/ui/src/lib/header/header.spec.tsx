import { render } from '@testing-library/react';

import Header from './header';

describe('Header', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<Header title="Dashboard"  backgroundColor="bg-cyan-600"/>);
    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
