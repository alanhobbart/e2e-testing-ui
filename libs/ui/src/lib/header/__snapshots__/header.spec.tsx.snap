// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render successfully 1`] = `
<body>
  <div>
    <header
      class="shadow bg-cyan-600"
    >
      <div
        class=""
      >
        <nav
          class="w-full md:flex-row md:flex-nowrap md:justify-start flex items-center p-4"
        >
          <div
            class="w-full mx-autp items-center flex justify-between md:flex-nowrap flex-wrap px-4"
          >
            <a
              class="text-sm text-white uppercase hidden lg:inline-block font-semibold"
              href="#pablo"
            >
              Dashboard
            </a>
          </div>
        </nav>
      </div>
    </header>
  </div>
</body>
`;
